CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 3.1.0
+* NEW: Added TypeScript declaration file for compatibility with TypeScript projects. See: https://github.com/getsentry/raven-js/pull/610
```

Comment:
There are obviously more changes here.

My first impression was "why are we doing a big version bump just for adding TS declarations", then realized there are lots of things.